js/compressed-token/src/actions/transfer.ts

```diff
@@ -6,30 +6,56 @@ import {
   TransactionSignature,
   ComputeBudgetProgram,
 } from '@solana/web3.js';
-import { CompressedTokenProgram } from '../program';
 import {
   CompressedProof_IdlType,
-  TlvDataElement_IdlType,
-  Utxo_IdlType,
   bn,
   defaultTestStateTreeAccounts,
   sendAndConfirmTx,
 } from '@lightprotocol/stateless.js';
 import { buildAndSignTx } from '@lightprotocol/stateless.js';
 import { BN } from '@coral-xyz/anchor';
 import { createTransferInstruction } from '../instructions';
-import { TokenTlvData_IdlType, TokenTransferOutUtxo_IdlType } from '../types';
+import { TokenTransferOutUtxo_IdlType } from '../types';
 import { getSigners } from './mint-to';
+import {
+  UtxoWithParsedTokenTlvData,
+  getCompressedTokenAccountsFromMockRpc,
+} from '../token-serde';
 
 /**
- * Transfer compressed tokens from one address to another
+ * @internal
+ *
+ * Selects the minimal number of compressed token accounts for a transfer
+ * 1. Sorts the accounts by amount in descending order
+ * 2. Accumulates the amount until it is greater than or equal to the transfer
+ *    amount
+ */
+function pickMinCompressedTokenAccountsForTransfer(
+  accounts: UtxoWithParsedTokenTlvData[],
+  transferAmount: BN,
+): UtxoWithParsedTokenTlvData[] {
+  let accumulatedAmount = bn(0);
+  const selectedAccounts = [];
+  accounts.sort((a, b) => b.parsed.amount.cmp(a.parsed.amount));
+  for (const account of accounts) {
+    if (accumulatedAmount.gte(bn(transferAmount))) break;
+    accumulatedAmount = accumulatedAmount.add(account.parsed.amount);
+    selectedAccounts.push(account);
+  }
+  if (accumulatedAmount.lt(bn(transferAmount))) {
+    throw new Error('Not enough balance for transfer');
+  }
+  return selectedAccounts;
+}
+/**
+ * Transfer compressed tokens from one owner to another
  *
  * @param connection     Connection to use
  * @param payer          Payer of the transaction fees
  * @param mint           Mint of the compressed token
  * @param amount         Number of tokens to transfer
- * @param currentOwner   Source address
- * @param newOwner       Destination address
+ * @param owner          Owner of the compressed tokens
+ * @param toAddress      Destination address of the recipient
```

Comment:
I think address is confusing because we use address as pda address.
What about toOwner or toPubkey?